main.go

```diff
@@ -619,3 +607,26 @@ func configFilePath() string {
 	}
 	return filepath.Join(fp, "mark")
 }
+
+func setLogLevel(cCtx *cli.Context) error {
+	logLevel := cCtx.String("log-level")
+	switch logLevel {
+	case "TRACE":
```

Comment:
can we use String() https://github.com/kovetskiy/lorg/blob/master/level.go#L27 instead of hardcoding those values here?